src/main/kotlin/com/salesforce/intellij/gateway/connector/AuthRedirectResponseEp.kt

```diff
@@ -0,0 +1,10 @@
+package com.salesforce.intellij.gateway.connector
+
+import com.intellij.openapi.extensions.ExtensionPointName
+import com.salesforce.pomerium.AuthenticationRedirectResponseHandler
+
+object AuthRedirectResponseEp {
```

Comment:
```suggestion
object AuthRedirectResponseHandlerEp {
```

nit: For better readability